source/_posts/2018-10-26-Going-to-ServerSide-swift-2018.md

```diff
@@ -0,0 +1,57 @@
+
+The whole Vapor team at Nodes met up in Berlin to participate in the conference and with this blog post we wanted to share a couple of highlights from the conference as well as giving some feedback on the conference in case any readers are considering to go next year.
+
+## Quick facts
+Some quick facts about ServerSide.swift 2018:
+- September 12-14 in Berlin (Adlershof)
+- One day of workshops. Four half-day workshops to choose from where two ran in parallel before lunch and the last two ran in parallel after lunch. This meant that you would be able to attend two in total.
+- Two days of talks. 8 talks per day and 18 speakers in total.
+- September 13 had a panel discussion after the last talk with five people from the community in the panel.
+- Website: [http://serversideswift.info](http://serversideswift.info) - videos should be available soon at the time of writing this post.
+
+## Talk highlights
+With 16 talks in total there has been a lot of knowledge to digest. Here are some highlights from the different talks we heard:
+
+### Vapor State Of The Union (Tanner Nelson)
+Since we at Nodes spend a lot of time using Vapor we were of course interested in the state of the union talk by Vapor's founder, Tanner Nelson. The talk went through the biggest changes from Vapor 2 to Vapor 3, being 1) how we deal with configs (read: no more JSON files), 2) how codable removes a lot of boiler plate code for us and 3) how Vapor is now built on top of SwiftNIO, making it possible for the Vapor team to delete a lot of code. The talk also gave a quick overview of the different packages that exist within the Vapor ecosystem and it provided a sneakpeak into what can be expected from Vapor Cloud 2 which [launched in alpha recently](https://medium.com/vapor-cloud/vapor-cloud-2-public-alpha-2d9ee14d0688). Last, the talk also touched upon the [Swift Server Work Group](https://swift.org/server/) and the [Swift Forums](https://forums.swift.org/c/development/server).
+
+### From Idea to Production at ING (Ian Partridge and Pim Stolk)
+It is always a good idea to have the finger on the pulse regarding alternative tools, and this split venture talk from Pim from [ING](https://www.ing.com) (Dutch bank) and Ian from [IBM and Kitura](https://www.kitura.io) provided just that.
+
+ING is a bank in The Netherlands where Kitura is playing a big role in their tech stack. The first part of the talk was given by Pim from ING and, besides presenting ING to the crowd, it revolved around the thoughts behind choosing Kitura, bringing innovative solutions into the banking world.
+
+Ian's talk focused on Kitura, an alternative framework to Vapor, and presented how the latest version has evolved from `2.0` to current `2.5`. A key takeaway was how they managed to integrate [OpenAPI](https://www.openapis.org/) in Kitura by automating their API documentation using auto-generated Swagger documentation and examples. This is a feature that would be welcomed by any API developer.
```

Comment:
Should probably mention " an alternative framework to Vapor,"  earlier, since the previous sentence already talked about Kitura